app/models/alchemy/page/page_natures.rb

```diff
@@ -33,6 +33,15 @@ def contains_feed?
       definition["feed"]
     end
 
+    def scoped?
+      definition["editable_by"]
+    end
+
+    def scoped_user
+      return unless scoped?
+      definition["editable_by"].split(', ')
```

Comment:
Please make usage of YAML and don't split a string. 

Multiple editor roles should be defined as:

```yaml
editable_by:
  - content_managers
  - reviewers
```